### DISCUSSION TITLE
Question about communication between users and developers

### DISCUSSION THREAD

**Discussion by UserGoodtm** [snipe/snipe-it#9218] [General]
I apologize for the possibly inappropriate and stupid question. Why do developers answer some bug reports more than once, and some questions are not answered? There are problems that stretch from version to version, but for one reason or another they are not fixed. For example, it took a long time to solve the problem (perhaps even now advanced search does not work very well) of advanced search. Does advanced search for custom fields work? I have long found a problem that occurs when deleting users, when the user ID number and the location ID number match. At the same time, I wrote a bug report by mistake, but the developers did not respond, the error was not corrected. Already version 5.1.2, and the error is in place. I would like more feedback from developers on emerging bug reports. Thanks for understanding. 
I really like the program and would like its development and stability.

**Comment by snipe** [comment]
> There are problems that stretch from version to version, but for one reason or another they are not fixed. 

We get to things as we can. We have and endless pool of bug reports and feature requests that never, ever ends. If a bug catches our attention, we try to get to it. If it's affecting paying customers, we try to escalate it and handle it quickly. (We have to pay our bills...) When we have a moment, we try to clean up the bug backlog as best we can. 

There's no big conspiracy theory here... We release often, and obviously not every bug can get fixed in every release. 

>  I have long found a problem that occurs when deleting users, when the user ID number and the location ID number match

That was fixed, per the commit message on the PR (#9233). 

You also h ave a habit of opening 35 bug reports in a day, without so much as a thank you, and frankly, it comes off a bit rude and entitled. That doesn't affect whether or not we handle bug reports, but it makes us less excited to do so when it's nothing but criticism and not so much as a thanks.
